Modules/_zoneinfo.c

```diff
@@ -36,8 +36,8 @@ typedef struct {
     PyObject *key;
     PyObject *file_repr;
     PyObject *weakreflist;
-    unsigned int num_transitions;
-    unsigned int num_ttinfos;
+    Py_ssize_t num_transitions;
```

Comment:
I opted to change these to `Py_ssize_t` to avoid having to deal downstream with precision.